src/components/forms/ConsentForm.js

```diff
@@ -62,42 +63,67 @@ const Consent = compose(
+                NIH Security Best Practices for Controlled-Access Data
+              </ExternalLink>
+              such that only authorized individuals possess access to data files;
             </li>
             <li>
-              You will not submit for publication or any other form of public dissemination analyses
-              or other reports on work using or referencing NIH datasets prior to an existing
-              embargo release date listed for the dataset (or dataset version) on dbGaP;
+              You acknowledge Intellectual Property Policies should they exist as specified in a dataset’s associated 
+              Data Use Certification; and,
             </li>
             <li>
-              You acknowledge the Intellectual Property Policies as specified in a dataset’s
-              associated Data Use Certification; and,
-            </li>
-            <li>
-              You will report any inadvertent data release in accordance with the terms in the Data
-              Use Certification, breach of data security, or other data management incidents
-              contrary to the terms of data access.
+              You will report any inadvertent data release in accordance with the terms in the Data Use Certification, 
+                breach of data security, or other data management incidents contrary to the terms of data access.
             </li>
           </ol>
+          <p>
+            Kids First DRC terms and conditions may be changed at any time via a public posting of revisions
+            to the Kids First DRC Website, Portal and/or Services. As a user, you agree to review the 
+            Terms & Conditions and Privacy Policies each time you use the Kids First DRC Website, Portal
+            and/or Services so that you are aware of any modifications made to these policies. By accessing
+            or using the Kids First DRC Website, Portal and/or Services, you agree with and to be bound by all
+            of the terms and conditions and policies as posted on the Kids First DRC Website, Portal
+            and/or Services at the time of your access or use, including the Privacy Policies then in effect.
+          </p>   
+          <p>
+            For documents and/or data available from DRC Services, the Kids First DRC does not warrant or assume
+            any legal liability or responsibility for the accuracy, completeness, or usefulness of any information,
+            apparatus, product, or process. No specific medical advice is provided by any Services, and Kids First
+            urges users of Services to consult with a qualified physician for diagnosis and for answers to personal
+            questions.              
+          </p>
+          <p>
+            If you have any questions about these terms, conditions or the practices of this site or any of the other
+            Kids First DRC Services, please contact us at support@kidsfirstdrc.org. For a full list of Kids first DRC
```

Comment:
I changed this a bit: `The full list of Kids first DRC policies are located at https://kidsfirstdrc.org/policies/.`

Reply:
done